packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -737,19 +737,20 @@ class FlutterError extends Error with DiagnosticableTreeMixin implements Asserti
   }
 }
 
-/// Dump the current stack to the console using [debugPrint] and
+/// Dump the stack to the console using [debugPrint] and
 /// [FlutterError.defaultStackFilter].
 ///
-/// The current stack is obtained using [StackTrace.current].
+/// If `stackTrace` is null, dumps the current stack using [StackTrace.current].
 ///
 /// The `maxFrames` argument can be given to limit the stack to the given number
-/// of lines. By default, all non-filtered stack lines are shown.
+/// of non-filtered lines. By default, all non-filtered stack lines are shown.
```

Comment:
nit: I'm not sure it helps to add `non-filtered` in here - I think I'd probably rather remove it from the other part of the sentence, since we don't really have any options for filtering on this method anyway.

Reply:
Without it, I think it's unclear how filtering and `maxFrames` interact with each other.  Is filtering done first, and then `maxFrames` frames are printed?  Or is the `StackTrace` truncated first and then filtered?

Reply:
I reworded the comment hopefully to both our liking.